platforms/common/cpp/utils/AbstractHttpSupport.cpp

```diff
@@ -737,9 +735,9 @@ StackSlot AbstractHttpSupport::deleteAppCookies(RUNNER_ARGS)
 
 ResponseEncoding AbstractHttpSupport::GetResponseEncodingFromString(std::string str)
 {
-    if (str == "utf8_js")
-        return ResponseEncodingUTF8js;
-    else if (str == "utf8") {
+    if (str == "utf8")
+        return ResponseEncodingWTF8;
+    else if (str == "wtf8") {
         return ResponseEncodingUTF8;
```

Comment:
Do we need to change the values ​​under these conditions (utf8 and wtf8)?

Reply:
Thanks. Fixed.